`npm install jsdom` with contextify fails using iojs@2.0.0

Here is the output

```bash
$ node --version
v2.0.0
$ npm --version
2.9.0
$ nvm --version
0.23.3
$ npm install 
npm WARN optional dep failed, continuing fsevents@0.3.5
> contextify@0.1.13 install /home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify
> node-gyp rebuild
make: Entering directory `/home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify/build'
  CXX(target) Release/obj.target/contextify/src/contextify.o
In file included from ../node_modules/nan/nan_new.h:181:0,
                 from ../node_modules/nan/nan.h:74,
                 from ../src/contextify.cc:3:
../node_modules/nan/nan_implementation_12_inl.h: In static member function ‘static NanIntern::FactoryBase<v8::Signature>::return_t NanIntern::Factory<v8::Signature>::New(NanIntern::Factory<v8::Signature>::FTH, int, NanIntern::Factory<v8::Signature>::FTH*)’:
../node_modules/nan/nan_implementation_12_inl.h:172:76: error: no matching function for call to ‘v8::Signature::New(v8::Isolate*, NanIntern::Factory<v8::Signature>::FTH&, int&, NanIntern::Factory<v8::Signature>::FTH*&)’
../node_modules/nan/nan_implementation_12_inl.h:172:76: note: candidate is:
/home/travis/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:27: note: static v8::Local<v8::Signature> v8::Signature::New(v8::Isolate*, v8::Handle<v8::FunctionTemplate>)
/home/travis/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:27: note:   candidate expects 2 arguments, 4 provided
make: *** [Release/obj.target/contextify/src/contextify.o] Error 1
make: Leaving directory `/home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/travis/.nvm/versions/io.js/v2.0.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:169:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1009:12)
gyp ERR! System Linux 2.6.32-042stab094.7
gyp ERR! command "/home/travis/.nvm/versions/io.js/v2.0.0/bin/iojs" "/home/travis/.nvm/versions/io.js/v2.0.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify
gyp ERR! node -v v2.0.0
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok 
npm WARN engine xmlbuilder@2.4.6: wanted: {"node":"0.8.x || 0.10.x || 0.11.x  || 1.0.x"} (current: {"node":"2.0.0","npm":"2.9.0"})
npm WARN prefer global jshint@2.7.0 should be installed with -g
npm ERR! Linux 2.6.32-042stab094.7
npm ERR! argv "/home/travis/.nvm/versions/io.js/v2.0.0/bin/iojs" "/home/travis/.nvm/versions/io.js/v2.0.0/bin/npm" "install"
npm ERR! node v2.0.0
npm ERR! npm  v2.9.0
npm ERR! code ELIFECYCLE
npm ERR! contextify@0.1.13 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the contextify@0.1.13 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/catberry/catberry/npm-debug.log
The command "eval npm install " failed. Retrying, 2 of 3.
> contextify@0.1.13 install /home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify
> node-gyp rebuild
make: Entering directory `/home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify/build'
  CXX(target) Release/obj.target/contextify/src/contextify.o
In file included from ../node_modules/nan/nan_new.h:181:0,
                 from ../node_modules/nan/nan.h:74,
                 from ../src/contextify.cc:3:
../node_modules/nan/nan_implementation_12_inl.h: In static member function ‘static NanIntern::FactoryBase<v8::Signature>::return_t NanIntern::Factory<v8::Signature>::New(NanIntern::Factory<v8::Signature>::FTH, int, NanIntern::Factory<v8::Signature>::FTH*)’:
../node_modules/nan/nan_implementation_12_inl.h:172:76: error: no matching function for call to ‘v8::Signature::New(v8::Isolate*, NanIntern::Factory<v8::Signature>::FTH&, int&, NanIntern::Factory<v8::Signature>::FTH*&)’
../node_modules/nan/nan_implementation_12_inl.h:172:76: note: candidate is:
/home/travis/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:27: note: static v8::Local<v8::Signature> v8::Signature::New(v8::Isolate*, v8::Handle<v8::FunctionTemplate>)
/home/travis/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:27: note:   candidate expects 2 arguments, 4 provided
make: *** [Release/obj.target/contextify/src/contextify.o] Error 1
make: Leaving directory `/home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/travis/.nvm/versions/io.js/v2.0.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:169:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1009:12)
gyp ERR! System Linux 2.6.32-042stab094.7
gyp ERR! command "/home/travis/.nvm/versions/io.js/v2.0.0/bin/iojs" "/home/travis/.nvm/versions/io.js/v2.0.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify
gyp ERR! node -v v2.0.0
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok 
npm ERR! Linux 2.6.32-042stab094.7
npm ERR! argv "/home/travis/.nvm/versions/io.js/v2.0.0/bin/iojs" "/home/travis/.nvm/versions/io.js/v2.0.0/bin/npm" "install"
npm ERR! node v2.0.0
npm ERR! npm  v2.9.0
npm ERR! code ELIFECYCLE
npm ERR! contextify@0.1.13 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the contextify@0.1.13 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/catberry/catberry/npm-debug.log
The command "eval npm install " failed. Retrying, 3 of 3.
> contextify@0.1.13 install /home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify
> node-gyp rebuild
make: Entering directory `/home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify/build'
  CXX(target) Release/obj.target/contextify/src/contextify.o
In file included from ../node_modules/nan/nan_new.h:181:0,
                 from ../node_modules/nan/nan.h:74,
                 from ../src/contextify.cc:3:
../node_modules/nan/nan_implementation_12_inl.h: In static member function ‘static NanIntern::FactoryBase<v8::Signature>::return_t NanIntern::Factory<v8::Signature>::New(NanIntern::Factory<v8::Signature>::FTH, int, NanIntern::Factory<v8::Signature>::FTH*)’:
../node_modules/nan/nan_implementation_12_inl.h:172:76: error: no matching function for call to ‘v8::Signature::New(v8::Isolate*, NanIntern::Factory<v8::Signature>::FTH&, int&, NanIntern::Factory<v8::Signature>::FTH*&)’
../node_modules/nan/nan_implementation_12_inl.h:172:76: note: candidate is:
/home/travis/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:27: note: static v8::Local<v8::Signature> v8::Signature::New(v8::Isolate*, v8::Handle<v8::FunctionTemplate>)
/home/travis/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:27: note:   candidate expects 2 arguments, 4 provided
make: *** [Release/obj.target/contextify/src/contextify.o] Error 1
make: Leaving directory `/home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/travis/.nvm/versions/io.js/v2.0.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:169:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1009:12)
gyp ERR! System Linux 2.6.32-042stab094.7
gyp ERR! command "/home/travis/.nvm/versions/io.js/v2.0.0/bin/iojs" "/home/travis/.nvm/versions/io.js/v2.0.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/travis/build/catberry/catberry/node_modules/jsdom/node_modules/contextify
gyp ERR! node -v v2.0.0
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok 
npm ERR! Linux 2.6.32-042stab094.7
npm ERR! argv "/home/travis/.nvm/versions/io.js/v2.0.0/bin/iojs" "/home/travis/.nvm/versions/io.js/v2.0.0/bin/npm" "install"
npm ERR! node v2.0.0
npm ERR! npm  v2.9.0
npm ERR! code ELIFECYCLE
npm ERR! contextify@0.1.13 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the contextify@0.1.13 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/catberry/catberry/npm-debug.log
The command "eval npm install " failed 3 times.
The command "npm install " failed and exited with 1 during .
Your build has been stopped.
```

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     ],
     "dependencies": {
         "bindings" : "*",
-        "nan" : "~1.7.0"
+        "nan" : "~1.8.4"
     },
     "devDependencies": {
         "nodeunit" : ">=0.5.x"

